### PR TITLE
Add 2022 values for CA use tax and income tax parameters

### PR DESCRIPTION
**Pull request by martinholmer**
Add CA tax policy parameters for 2022.

### CODE REVIEW

**Comment by martinholmer** [comment]
@nikhilwoodruff,  why does the test_cps fail on Windows, but not on Mac (local) or Linux (GitHub)?

**Comment by nikhilwoodruff** [comment]
> @nikhilwoodruff,  why does the test_cps fail on Windows, but not on Mac (local) or Linux (GitHub)?

On mobile so will take a look in a bit, but IIRC could it be because of HDF5 package differences in Windows?

**Comment by martinholmer** [comment]
@nikhilwoodruff said:
> could it be because of HDF5 package differences in Windows?

This error just started appearing this morning, so I don't see how that is an explanation.

**Comment by nikhilwoodruff** [comment]
Martin- you're right. The last PR merged was the first to see this issue, but it definitely didn't cause it because it only changed a parameter value. I think something changed in either the GitHub Windows environment or one of the dependencies. Looking into it

**Comment by nikhilwoodruff** [comment]
@martinholmer think it's fixed now

**Comment by martinholmer** [comment]
@nikhilwoodruff said:
> think it's fixed now

Yes, it seems so.  Thanks!

**Comment by martinholmer** [comment]
@MaxGhenis, PR #1874 is now ready for your review.

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+  2022-01-01: 120
 
 metadata:
   period: year
   unit: currency-USD
   breakdown:
   - filing_status
-  name: ca_renters_credit_amount
-  label: "California renters tax credit amount"
+  name: ca_renter_credit_amount
```
[the remark this answers is not in the document]
Comment (martinholmer) [reply policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml]: Done

File: policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
+  label: "California renter tax credit amount"
   reference:
-  - title: Cal. Rev. & Tax. Code § 17053.5
```
[the remark this answers is not in the document]
Comment (martinholmer) [reply policyengine_us/parameters/gov/states/ca/tax/income/credits/renter/amount.yaml:22]: Restored

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**